packages/textlint-tester/src/textlint-tester.ts

```diff
@@ -40,20 +40,32 @@ function assertHasFixer(ruleCreator: any, ruleName: string): any {
     throw new Error(`Not found \`fixer\` function in the ruleCreator: ${ruleName}`);
 }
 
+export type TestConfig = {
+    plugins: { pluginId: string; plugin: TextlintPluginCreator }[];
+    rules: { ruleId: string; rule: TextlintRuleCreator }[];
+};
+
+function isTestConfig(arg: any): arg is TestConfig {
+    return arg.plugins !== undefined;
```

Comment:
`rules` is required at least one.

- is `rules` array
- is `rules` length > 0
   - assert: rule item should have `ruleId` and `rule`
- is `plugins` array
   - assert: Each plugin item should have `pluginId` and `plugin`

It is better that assert item object.
It is helpful for (non-TypeScript) user.

Reply:
It is good that export and  test it.

Patterns:

- OK: `rules` only
- OK `rules` and `plugins`
- NG: `plugins` only
- NG: Not have property